### ISSUE TITLE
wrong tagging

### ISSUE THREAD

**Issue by sajjad-shirazy** [nlp-compromise/compromise#372]
these are terms of `"They are flowers."`. `flowers` is tagged `Verb`
```
[  
   {  
      "text":"They",
      "normal":"they",
      "tags":[  
         "TitleCase",
         "Pronoun",
         "Noun",
         "Singular"
      ]
   },
   {  
      "text":"Are",
      "normal":"are",
      "tags":[  
         "Copula",
         "Verb",
         "VerbPhrase",
         "TitleCase"
      ]
   },
   {  
      "text":"flowers.",
      "normal":"flowers",
      "tags":[  
         "Verb",
         "VerbPhrase",
         "TitleCase"
      ]
   }
]
```

**Comment by spencermountain** [comment]
thanks, will check this out today.

**Comment by spencermountain** [comment]
fixed in 10.5.0 :rocket:
